jobs/standalone/templates/bin/ctl

```diff
@@ -11,6 +11,8 @@ case $1 in
     sysctl vm.overcommit_memory=1
     echo never > /sys/kernel/mm/transparent_hugepage/enabled
 
+    chpst -u vcap:vcap redis-server --version > /var/vcap/store/standalone/REDIS_VERSION
```

Comment:
What is the purpose of the `chpst` invocation of `redis-server`?  Since the shell redirection happens before the chpst process is spun up, the file will still be owned by root:root.  Does running redis-server as a non-vcap user muck up some internal state?

Reply:
When a service starts up initially, it may try to create files like sockets or logs.  Although I just want to get the Version here, running as VCAP should ensure any side-effects will happen as VCAP and not cause issues further down.  Logs getting created as root for example.

Redis-server shouldn't have side effects but trying to use a pattern here that will be safe for other forges to use to record the Version string.

Reply:
Updated to chown the two version files.  The chst will ensure any side effects will be as VCAP as desired